deepquest/utils/callbacks.py

```diff
@@ -474,15 +476,11 @@ def evaluate(self, epoch, counter_name='epoch', logs={}):
             if self.metric_name:
                 self.model_to_eval.plot(counter_name, set(all_metrics), self.set_name, upperbound=self.max_plot)
 
-        # Save the model
-        if self.save_each_evaluation:
-            from keras_wrapper.cnn_model import saveModel
-            saveModel(self.model_to_eval, epoch, store_iter=not self.eval_on_epochs)
-
```

Comment:
This removes the model save here since it is already saved by `on_epoch_end` which was resulting in the model being saved twice every epoch. @fredblain are you aware why this may be incorrect or why we may need extra parameters to account for this kind of behaviour, there are a lot of parameters..